erikfuego/im-hangry
# Created simple gin hello world app

This introduces a basic hello world application using the Gin framework

Testing steps:
- run the application with `go run .` from the root folder
- go to `localhost:8080/hello-world` and you will see a JSON response that says "hello world"
